Bmx.Idp.Okta/OktaClient.cs

```diff
@@ -13,7 +15,16 @@ public class OktaClient : IIdentityProvider {
 		private readonly JsonSerializerOptions _serializeOptions =
 			new JsonSerializerOptions {PropertyNamingPolicy = JsonNamingPolicy.CamelCase,};
 
-		private static readonly HttpClient HttpClient = new HttpClient();
+		private static readonly CookieContainer CookieContainer = new CookieContainer();
+
+		private static readonly HttpClient HttpClient = new HttpClient( new HttpClientHandler {
+			CookieContainer = CookieContainer
+		} );
```

Comment:
Curious: why static? It seems like every instance of this class should have their own HttpClient.

Reply:
Good catch, there is no reason for it to be static since if there was somehow multiple instances they should have their own isolated context and cookies.